elements/pfe-accordion/BaseAccordion.scss

```diff
@@ -0,0 +1,5 @@
+:host {
+    transition-property: box-shadow, border;
```

Comment:
fascinating! how'd you arrive at this solution?

Reply:
Deconstructing and reconstructing the pre-pfe1:1 accordion and figuring out what was left that made it work alongside the demos at https://www.w3.org/WAI/ARIA/apg/example-index/accordion/accordion gave me a good amount of working examples to figure out what was causing the voiceover to work